src/test/test-string-util.c

```diff
@@ -917,7 +917,7 @@ static void test_strverscmp_improved(void) {
                 "123-3.1",
                 "123^patch1",
                 "123^1",
-                "123.a-1"
+                "123.a-1",
```

Comment:
@yuwata I assume this was unintentional?

Otherwise we can wrap it in brackets to make coverity happy.